ci.sh

```diff
@@ -77,15 +77,20 @@ run_normal_tests() {
     RUST_BACKTRACE=1 cargo test $TEST_FEATURES --manifest-path ./e2e_tests/Cargo.toml normal_tests
 }
 
+run_old_e2e_tests() {
+    echo "Execute old end-to-end normal tests"
+    # The version of the Parsec client used in those old tests expect the socket to be at
+    # /tmp/parsec/parsec.sock. This can not be created in the Dockerfile as this is where
+    # the repository is checked out.
+    ln -s /tmp/parsec.sock /tmp/parsec/parsec.sock
+    RUST_BACKTRACE=1 cargo test --manifest-path /tmp/old_e2e_tests/Cargo.toml normal_tests
```

Comment:
Could you maybe raise an issue to identify a subset of tests that we could run (or create new ones just for this)? It's fine for now to run all of them, but really most of our tests verify functionality in the service, we probably don't need all of them here. The stability of that service functionality is tested in other places anyway.

Reply:
Yes, I understand your point. I mainly execute `normal_tests` for simplicity here instead of naming the subset of tests, since it takes less than two seconds to execute all of them anyway.

Reply:
Hmm, this might sound strange, but: can you compile them in the Dockerfile after copying? 😬

Reply:
Yeah I guess I could! For the per provider tests, the Rust compiler should not change at "run time"